lighthouse-core/test/gather/computed/metrics/lantern-first-cpu-idle-test.js

```diff
@@ -28,4 +29,20 @@ describe('Metrics: Lantern TTFCPUI', () => {
     assert.ok(result.optimisticGraph, 'should have created optimistic graph');
     assert.ok(result.pessimisticGraph, 'should have created pessimistic graph');
   });
+
+  describe('#getFirstCPUIdleWindowStart', () => {
+    it('should sort tasks', () => {
+      const tasks = new Map([
+        [{type: 'cpu'}, {startTime: 600, endTime: 700, duration: 100}],
+        [{type: 'cpu'}, {startTime: 300, endTime: 400, duration: 100}],
+        [{type: 'cpu'}, {startTime: 0, endTime: 100, duration: 100}],
+        [{type: 'cpu'}, {startTime: 100, endTime: 200, duration: 100}],
+        [{type: 'cpu'}, {startTime: 500, endTime: 600, duration: 100}],
+        [{type: 'cpu'}, {startTime: 200, endTime: 300, duration: 100}],
+        [{type: 'cpu'}, {startTime: 400, endTime: 500, duration: 100}],
+      ]);
+
+      assert.equal(LanternFCPUI.getFirstCPUIdleWindowStart(tasks, 0), 700);
+    });
+  });
```

Comment:
it might be harder to set up, but maybe a `Simulator` test that reads in artifacts and similarly asserts that `nodeTimings` are in `startTime` order (and wouldn't have been before this change)? I think our `stableSort` test for a trace has caught at least one regression that was accidentally reordering traces :)

Reply:
(or maybe there's an easier way to unit test it without calling private stuff; I didn't look very hard :)

Reply:
done